spec/features/upload_lead_image_spec.rb

```diff
@@ -12,6 +12,13 @@
     then_i_should_be_able_to_see_the_lead_image_on_the_summary_page
   end
 
+  scenario "User uploads a lead image and asset manager fails to upload it" do
```

Comment:
Perhaps we can move this into its own file? Content tagger has multiple scenarios per file, and it's not improving readability (we should write this down).

Reply:
I think that's fair.  I'm a little concerned that we've got so many test files in the features directory that it's becoming difficult to navigate, but that's probably a conversation we can have outside of this PR 😸